static/app/views/organizationLayout.tsx

```diff
@@ -0,0 +1,77 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {Alert} from 'sentry/components/alert';
+import HookOrDefault from 'sentry/components/hookOrDefault';
+import LoadingError from 'sentry/components/loadingError';
+import LoadingTriangle from 'sentry/components/loadingTriangle';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import Sidebar from 'sentry/components/sidebar';
+import {ORGANIZATION_FETCH_ERROR_TYPES} from 'sentry/constants';
+import {t} from 'sentry/locale';
+import OrganizationStore from 'sentry/stores/organizationStore';
+import {useLegacyStore} from 'sentry/stores/useLegacyStore';
+import {space} from 'sentry/styles/space';
+
+interface OrganizationLayoutProps {
+  /**
+   * Render the sidebar when possible if enabled
+   */
+  includeSidebar: boolean;
+  children?: React.ReactNode;
```

Comment:
```suggestion
  children: React.ReactNode;
```
It's a good idea to explicitly type children as things like Layouts components and Providers should not be expected to be used without them. You could also use something like `NonNullable<React.ReactNode>` since we like don't want expect null children